builder/vsphere/clone/step_customize.go

```diff
@@ -136,6 +139,10 @@ func (c *CustomizeConfig) Prepare() []error {
 		options_number = options_number + 1
 	}
 	if c.WindowsSysPrepFile != "" {
+		warnings = append(warnings, "`windows_sysprep_file` is deprecated and will be removed in a future release. please use `windows_sysprep_text`.")
```

Comment:
Do we need a mutual exclusivity check here to error if both `windows_systprep_file` and `windows_sysprep_text` are provide?

Reply:
Agree! Otherwise it conflicts.

Reply:
A mutual exclusivity check is already included here. Line 149 prints an error message if options_number > 1 which is true if both windows_sysprep_file and windows_sysprep_text are included.

Reply:
Thanks for calling this out. I didn't notice that logic above.